### PR TITLE
AMB-980 Get Patient from PDS

### PR DESCRIPTION
**Pull request by artronics**
Add mediation service

## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -45,9 +45,7 @@ _dist_include="pytest.ini poetry.lock poetry.toml pyproject.toml Makefile build/
 release: clean publish build-proxy
 	mkdir -p dist
 	for f in $(_dist_include); do cp -r $$f dist; done
-	cp ecs-proxies-deploy.yml dist/ecs-deploy-sandbox.yml
-	cp ecs-proxies-deploy.yml dist/ecs-deploy-internal-qa-sandbox.yml
-	cp ecs-proxies-deploy.yml dist/ecs-deploy-internal-dev-sandbox.yml
+	cp ecs-proxies-deploy.yml dist/ecs-deploy-internal-dev.yml
```
Comment (artronics) [inline Makefile:48]: Add `int` env to this.

File: mediation_service/mediation/main.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+from starlette.responses import Response
+from starlette.status import HTTP_200_OK
+
+app = FastAPI()
+
+
+@app.get("/_status")
+def status():
+    return Response(status_code=HTTP_200_OK)
+
```
Comment (artronics) [inline mediation_service/mediation/main.py:12]: Add Alllergy endpoint. See the AMB-1001 ticket

**Review by hirra-farooq — DISMISSED** [review]

**Review by Valentinolucas — APPROVED** [review]

**Pull request merged**